task.go

```diff
@@ -251,8 +251,19 @@ func (t *task) Resize(ctx context.Context, w, h uint32) error {
 	return err
 }
 
+const (
+	NoPivotRoot              = "io.runc.option.no-pivot-root"
```

Comment:
Those are linux specific.

Any way we can avoid duplicating the definition between the client and the shim?

Also, shouldn't there be a `TaskOption` prefix or suffix? (this one being a nit)